src/manpage.js

```diff
@@ -1,3 +1,4 @@
+import { _window } from "../lib/window.js";
```

Comment:
Given that the manpage code should really only load in browser context, we shouldn't need this import

Reply:
What do you suggest? Shall we override window in NodeJS, when window is undefined?

Reply:
The other way around: if we're in node, we do nothing in this file (i.e. we run the code in a conditional block), and similarly, we don't run the `Font.manPage = manPage;` binding if we're in node context.